server/src/main/java/org/elasticsearch/search/runtime/AbstractLongScriptFieldQuery.java

```diff
@@ -20,6 +21,15 @@
  */
 abstract class AbstractLongScriptFieldQuery extends AbstractScriptFieldQuery<AbstractLongFieldScript> {
 
+    AbstractLongScriptFieldQuery(
+        Script script,
+        String fieldName,
+        Query approximation,
+        Function<LeafReaderContext, AbstractLongFieldScript> scriptContextFunction
+    ) {
+        super(script, fieldName, approximation, scriptContextFunction);
+    }
+
```

Comment:
I'm not usually a fan of having two ctors for stuff like this, but this change is long enough already so I cut my losses.